gateway/ln-gateway/src/client.rs

```diff
@@ -60,7 +53,8 @@ impl StandardGatewayClientBuilder {
 
         let mut client_builder = ClientBuilder::default();
         client_builder.with_module_gens(registry);
-        client_builder.with_primary_module(1);
```

Comment:
@m1sterc001guy the original code seems to set the dummy module as the primary module, no?